docs/integrations/sources/zoho-crm.md

```diff
@@ -0,0 +1,128 @@
+
+Airbyte uses [REST API](https://www.zoho.com/crm/developer/docs/api/v2/modules-api.html) to fetch data from Zoho CRM.
+
+This Source Connector is based on an [Airbyte CDK](https://docs.airbyte.io/connector-development/cdk-python).
+
+### Output schema
+
+This Source is capable of syncing:
+
+* standard modules available in Zoho CRM account
+* custom modules manually added by user, available in Zoho CRM account
+* custom fields in both standard and custom modules, available in Zoho CRM account
+
+The discovering of Zoho CRM module schema is made dynamically based on Metadata API and should generally take no longer than 10 to 30 seconds.
+
+### Note:
+
+Some of Zoho CRM Modules may not be available for sync due to limitations of Zoho CRM Edition or permissions scope. For details refer to the [Scopes](https://www.zoho.com/crm/developer/docs/api/v2/scopes.html) section in the Zoho CRM documentation.
+
+### Data type mapping
+
+| Integration Type      | Airbyte Type | Notes                     |
+|:----------------------|:-------------|:--------------------------|
+| `boolean`             | `boolean`    |                           |
+| `double`              | `number`     |                           |
+| `currency`            | `number`     |                           |
+| `integer`             | `integer`    |                           |
+| `profileimage`        | `string`     |                           |
+| `picklist`            | `string`     | enum                      |
+| `textarea`            | `string`     |                           |
+| `website`             | `string`     | format: uri               |
+| `date`                | `string`     | format: date              |
+| `datetime`            | `string`     | format: date-time         |
+| `text`                | `string`     |                           |
+| `phone`               | `string`     |                           |
+| `bigint`              | `string`     | airbyte_type: big_integer |
+| `event_reminder`      | `string`     |                           |
+| `email`               | `string`     | format: email             |
+| `autonumber`          | `string`     | airbyte_type: big_integer |
+| `jsonarray`           | `array`      |                           |
+| `jsonobject`          | `object`     |                           |
+| `multiselectpicklist` | `array`      |                           |
+| `lookup`              | `object`     |                           |
+| `ownerlookup`         | `object`     |                           |
+| `RRULE`               | `object`     |                           |
+| `ALARM`               | `object`     |                           |
+
+Any other data type not listed in the table above will be treated as `string`.
+
+### Features
+
+| Feature                                   | Supported? \(Yes/No\) | Notes |
```

Comment:
If we didn’t have notes we can delete `Notes` column.

Reply:
done